dmcontent/govuk_frontend.py

```diff
@@ -141,6 +178,19 @@ def _params(
     if data and question.id in data:
         params["value"] = data[question.id]
 
+    # For question pages with multiple inputs, we use a fieldset.
+    # At the moment, list is the only applicable question type,
+    # but checkboxes, radios and multiquestions will probably join
+    # this check.
+    if question.get("type") == "list":
+        params["fieldset"] = {
+            "legend": {
+                "text": get_label_text(question),
+                "isPageHeading": True,
+                "classes": "govuk-fieldset__legend--l"
+            }
+        }
```

Comment:
I see the advantage of having a separate function `get_label_text` for the text part of a legend.

Can I suggest we also have a separate function `govuk_fieldset`that generates the parameters for the fieldset, rather than adding it to `_params`?

We can add the params to the list input params easily with `params["fieldset"] = govuk_fieldset(question)`, and we can worry about what to do with other components later.

Reply:
Done!